### DISCUSSION TITLE
AMT only working when device is powered on?

### DISCUSSION THREAD

**Discussion by gitwittidbit** [Ylianst/MeshCentral#6054] [Issues]
Hi,

I have had MeshCentral set up and running for a few weeks and I'm loving it.

Recently, I connected my first (and only) AMT device (Lenovo Thinkstation).

My issue is that MeshCentral only seems to see it, when the device is already on. But after it is shut down, MC reports that it was disconnected. Now, my primary reason for connecting the PC to MC is to be able to power it on remotely. And this is the only thing that doesn't seem to work...

I can reach AMT in my browser at any time. But MC can connect only, when the device is switched on.

Is this a misconfiguration on my part or is this expected behaviour?

Thanks!

**Comment by si458** [comment]
Are u sure u have connected the computer to the amt correctly?
This sounds to me that it's the meshagent you are using and not the AMT.
You can see if the amt is activated in The general tab or the details tab of the device and also what mode its setup in too

Also if u have multiple computers on the same network and in the same group in meshcentral,  so long as your device has wake on lan setup/activated, then meshcentral will tell other devices that are online, hey wake up this computer plz with wol packets!

**Comment by gitwittidbit** [comment]
Well, on the general tab for this particular machine, it says that it has recognized Intel AMT. And under the Intel AMT tab on this page, it shows a lot of AMT info that I can't see when I access AMT via the browser. So, yes, I'd say this connection is via AMT.

**Reply by si458** [reply]
can you share screenshots of the AMT info on the 2 tabs please?

**Comment by gitwittidbit** [comment]
of course:

![Screenshot 2024-04-25 at 20 58 57](https://github.com/Ylianst/MeshCentral/assets/83287661/2715e755-cdc3-46e0-90ef-582ee8d98451)

![Screenshot 2024-04-25 at 20 57 06](https://github.com/Ylianst/MeshCentral/assets/83287661/5fe2f0a7-9769-4619-aac4-9642e3b60d93)

**Reply by si458** [reply]
right so it is indeed connected via AMT but its connected via `AMT` and not `AMT Cira`
is your meshcentral server on the same network as your computer?
or is the meshcentral server "in the cloud" and remote?
if you go into your group settings, what is the `Intel AMT` Policy set as? (share screenshot if you can, also click the pencil)

**Comment by gitwittidbit** [comment]
MC is running in a local VM but on a different network from the AMT port of the machine. This is more by coincidence than purpose (because the network interfaces of the machine kept switching names). I could probably put the machine's AMT port on the same network as MC, if that helps.

Intel AMT is set to "No Policy"

**Reply by si458** [reply]
right, so because its set to `No Policy` basically meshcentral does NOTHING as it doesnt know anything about the computers AMT
so you have to give it details but this wont work all the time because the computer doesnt connect to meshcentral, only when its switched on!

one thing you can try it change the policy to `Fully Automatic` and let meshcentral reconfig the remote devices AMT to work with it

note, you will need to go into the Intel amt panel, connect to it (like ur second screen shot above), go into the network settings panel,
then change the Name & Domain, so it reads `COMPNAME.MESHCENTRALDOMAIN.COM`

also you might need to check your meshcentral is setup in WAN+LAN mode with a `cert` dns name set

https://ylianst.github.io/MeshCentral/intelamt/
